### ISSUE TITLE
Owncloud handels bad iCalendar files bad

### ISSUE THREAD

**Issue by mogoh** [owncloud/core#26319]
### Steps to reproduce
Import a bad iCalendar file in your OwnCloud calendar. Something like this:
```
BEGIN:VCALENDAR
PRODID:QIS-LSF HIS GmbH
VERSION:2.0
BEGIN:VTIMEZONE
TZID:Europe/Berlin
X-LIC-LOCATION:Europe/Berlin
BEGIN:DAYLIGHT
TZOFFSETFROM:+0100
TZOFFSETTO:+0200
TZNAME:CEST
DTSTART:19700329T020000
RRULE:FREQ=YEARLY;BYDAY=-1SU;BYMONTH=3
END:DAYLIGHT
BEGIN:STANDARD
TZOFFSETFROM:+0200
TZOFFSETTO:+0100
TZNAME:CET
DTSTART:19701025T030000
RRULE:FREQ=YEARLY;BYDAY=-1SU;BYMONTH=10
END:STANDARD
END:VTIMEZONE
METHOD:PUBLISH
BEGIN:VEVENT
DTSTART;TZID=Europe/Berlin:20161021T120000
DTEND;TZID=Europe/Berlin:20161021T140000
RRULE:FREQ=WEEKLY;UNTIL=20170210T235900Z;INTERVAL=1;BYDAY=FR
EXDATE:20161223T140000Z,20161230T140000Z,
DTSTAMP:20161009T172730Z
UID:180731242069
DESCRIPTION:
SUMMARY:Foo
END:VEVENT
END:VCALENDAR
```
Notice this line, which has a trailing comma:
```
EXDATE:20161223T140000Z,20161230T140000Z,
```

### Expected behaviour
OwnCloud should recognize that this is wrong.

### Actual behaviour
OwnCloud imports the data, but also safes the empty EXDATE. Thus is has Problems to display it. Also it can not synchronize this calendar any more.

ownCloud 8.2.7 (stable)

**Comment by RealRancor** [comment]
> ownCloud 8.2.7 (stable) 

There won't be any CalDAV/CardDAV support for this version. Please upgrade to oC 9.0.x and try to reproduce this again.

**Comment by PVince81** [comment]
@DeepDiver1975 didn't you add better validation recently ? I remember seeing a PR but can't seem to find it.

**Comment by mogoh** [comment]
I upgraded to Version 9.1.1 (as suggested). Now it seams that ownCloud handels this better. If I import a bad ics-file it doesn't appear and does not corrupt the database. So I thinks this can be closed. How ever a visual feedback that the import failed would be nice. Thank you for your help.

**Comment by DeepDiver1975** [comment]
@pvince81 I closed the pr in favor of upgrade to sabre 3.2
